ovos_persona_server/embeddings.py

```
@@ -0,0 +1,153 @@
+    total_tokens: int = prompt_tokens
+
+    try:
+        vectors: List[List[float]] = [embedder.get_embeddings(t) for t in texts_to_embed]
+    except Exception as e:
+        raise HTTPException(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+                            detail=f"Failed to generate embeddings: {e}") from e
+
+    # TODO - b64
+    if request_body.encoding_format == "base64":
+        raise HTTPException(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+                            detail=f"Failed to generate embeddings: base64 support not yet implemented")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)